jupyterhub/tests/selenium/test_browser.py

```diff
@@ -110,11 +127,9 @@ async def login(browser, user, pass_w):
     await webdriver_wait(browser, EC.url_changes(browser.current_url))
 
 
-async def test_submit_login_form(app, browser):
-    user = "test_user"
-    pass_w = "test_user"
+async def test_submit_login_form(app, browser, user="test_user", pass_w="test_user"):
```

Comment:
It doesn't make sense to have default values for arguments in test functions. Arguments to test functions are always fixture names.